lib/xudclient/XudClient.js

```diff
@@ -1,7 +1,7 @@
 const http = require('http');
 const assert = require('assert');
 
-class XUClient {
+class XudClient {
```

Comment:
I'd keep this as it was before. "Exchange Union Client" is what we should be going for, "Exchange Union Daemon Client" is overkill. This is in line with the naming scheme of bitcoind, btcd, and others.